src/js/components/Provenance/Tables/ExtraPropertiesTable.tsx

```diff
@@ -20,17 +20,17 @@ const ExtraPropertiesTable = ({ extraProperties }: ExtraPropertiesTableProps) =>
           dataIndex: 'values',
           render: (_, { values }) =>
             values.map((v, i) => (
-              <>
-                <LinkIfUrl key={i} text={v.value.toString()} />
+              <div key={i}>
```

Comment:
this breaks how multiple links are rendered - instead, use `<React.Fragment key={i}>{...}</React.Fragment>` or similar

Reply:
For multiple entries, I get this, which doesn't look broken: 

![Screenshot from 2024-03-21 09-17-41](https://github.com/bento-platform/bento_public/assets/18051642/709eeb3a-e053-45f6-aaa0-9f9c2cd91c2f)

Reply:
hmm I guess it doesn't render like I thought - i'm surprised the divs don't render as blocks but maybe there's some extra css at play. `<React.Fragment>` will still produce cleaner markup, since it will eliminate needless DOM elements.

Reply:
thanks, changed to `span`

Reply:
`<React.Fragment>` (or cleaner to `import { Fragment } from 'react'`) is still preferable, since it won't result in a `<span>` that isn't needed

Reply:
extra import vs a few spans? Anyway, changed to Fragment